humble/distribution.yaml

```diff
@@ -6375,6 +6375,12 @@ repositories:
       url: https://github.com/ros/resource_retriever.git
       version: humble
     status: maintained
+  reasoning_msgs:
```

Comment:
I think the problem here is that "reasoning" is too generic a term.  I'm going to suggest we rename this to `situational_graph_reasoning_msgs`, which will make this a lot easier to understand.

Reply:
@clalancette this will require some changes at the code level also, I will do it and come back with a new commit.